card/db/migrate_core_cards/20170719163708_update_bootswatch_themes.rb

```diff
@@ -1,63 +1,80 @@
 # -*- encoding : utf-8 -*-
 
 class UpdateBootswatchThemes < Card::Migration::Core
-  THEMES = %w[Cerulean Cosmo Cyborg Darkly Flatly Journal Litera Lumen Lux Materia Minty
-              Pulse Sandstone Simplex Slate Solar Spacelab Superhero United Yeti]
+  THEMES = %w[Cerulean Cosmo Cyborg Darkly Flatly Journal Litera Lumen Lux
+              Materia Minty Pulse Sandstone Simplex Slate Solar Spacelab
+              Superhero United Yeti]
              .map(&:downcase).freeze
 
   def up
     THEMES.each do |theme_name|
-      Card.exists?("#{theme_name} skin") ? update_skin(theme_name) : create_skin(theme_name)
+      Skin.new(theme_name).create_or_update
     end
+    Skin.new("Bootstrap default").update_scss file_name: "variables"
   end
 
-  def resource_path theme_name, resource
-    data_path "b4_themes/#{theme_name}/#{resource}"
-  end
+  class Skin
```

Comment:
It looks like there are a lot of non-renaming changes in here.  Are these part of getting the renaming to work, or are they unrelated?